src/libs/jira-lib.ts

```diff
@@ -135,6 +135,35 @@ function enhanceIssuesWithDescriptionText(issues: Issue[]): Issue[] {
   return issues.map(enhanceIssueWithDescriptionText);
 }
 
+function textToAdf(text: string): any {
+  if (!text) {
+    return {
+      type: "doc",
+      version: 1,
+      content: []
+    };
+  }
+
+  // Split text into paragraphs
+  const paragraphs = text.split('\n').filter(line => line.trim() !== '');
+  
+  const content = paragraphs.map(paragraph => ({
+    type: "paragraph",
+    content: [
+      {
+        type: "text",
+        text: paragraph
+      }
+    ]
+  }));
+
+  return {
+    type: "doc",
+    version: 1,
+    content: content
+  };
```

Comment:
minor readability tweak ... instead of content:content, I believe you can use the property shorthand here:
`return { type: "doc", version: 1, content };`